packages/drawer/package.json

```diff
@@ -16,7 +16,8 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "@material/drawer": "=4.0.0-canary.e851d4f40.0",
+    "@material/drawer": "=4.0.0-canary.22d7ad2fb.0",
+    "@material/list": "=4.0.0-canary.22d7ad2fb.0",
```

Comment:
Let's ping Abhinay to see if they can use = in their canary deps.

Reply:
PR is open for it https://github.com/material-components/material-components-web/pull/5182